pkg/recipes/terraform/config/testdata/providers-emptyazureconfig.tf.json

```diff
@@ -6,6 +6,12 @@
         "namespace": "radius-system",
         "secret_suffix": "test-secret-suffix"
       }
+    },
+    "required_providers": {
+      "azurerm": {
+        "source": "hashicorp/azurerm",
+        "version": "~> 2.0"
+      }
```

Comment:
Should we have a test with multiple required providers?

Reply:
Two tests with multiple require_providers exist: 
testdata/providers-overrideucpproviderconfig.tf.json
testdata/providers-valid.tf.json